sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/orc/OrcFileFormat.scala

```diff
@@ -26,7 +26,7 @@ import org.apache.hadoop.mapred.JobConf
 import org.apache.hadoop.mapreduce._
 import org.apache.hadoop.mapreduce.lib.input.FileSplit
 import org.apache.hadoop.mapreduce.task.TaskAttemptContextImpl
-import org.apache.orc._
+import org.apache.orc.{OrcUtils => _, _}
```

Comment:
Hey @srowen, I was going through the changes here. I am not sure, what this change means?

Reply:
Ah right, the issue is there is also an org.apache.org.OrcUtils, in addition to the one in this class's package. The code below wants to use the latter, the Spark OrcUtils. In 2.12, the local package class wins, but in 2.13 it doesn't, so the wrong one is imported.

This import syntax says "don't import OrcUtils from org.apache.orc, but import everything else"

Reply:
Yes. We should use `Spark's OrcUtils`.